<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuF">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrintPreview"/>
    <addaction name="actionPDF"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuE">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDel"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionReplace"/>
   </widget>
   <widget class="QMenu" name="menuH">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuV">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <addaction name="actionToobarShow"/>
    <addaction name="actionStatusBarShow"/>
    <addaction name="separator"/>
    <addaction name="actionHighLightShow"/>
    <addaction name="separator"/>
    <addaction name="actionzoomIn"/>
    <addaction name="actionzoomOut"/>
   </widget>
   <widget class="QMenu" name="menuO">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <addaction name="actionSwitch"/>
    <addaction name="separator"/>
    <addaction name="actionFont"/>
    <addaction name="actionFontColor"/>
    <addaction name="separator"/>
    <addaction name="actionleft"/>
    <addaction name="actioncenter"/>
    <addaction name="actionright"/>
    <addaction name="actionjustify"/>
   </widget>
   <widget class="QMenu" name="menuW">
    <property name="title">
     <string>窗口(&amp;W)</string>
    </property>
    <widget class="QMenu" name="menuViewMode">
     <property name="title">
      <string>视图模式</string>
     </property>
     <addaction name="actiontabbed"/>
     <addaction name="actionwindow"/>
    </widget>
    <widget class="QMenu" name="menuWindows">
     <property name="title">
      <string>选择窗口</string>
     </property>
    </widget>
    <addaction name="menuViewMode"/>
    <addaction name="separator"/>
    <addaction name="actionTile"/>
    <addaction name="actionCascade"/>
    <addaction name="separator"/>
    <addaction name="actionPrevious"/>
    <addaction name="actionNext"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
    <addaction name="separator"/>
    <addaction name="menuWindows"/>
   </widget>
   <addaction name="menuF"/>
   <addaction name="menuE"/>
   <addaction name="menuO"/>
   <addaction name="menuV"/>
   <addaction name="menuW"/>
   <addaction name="menuH"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionFind"/>
   <addaction name="separator"/>
   <addaction name="actionFont"/>
   <addaction name="actionFontColor"/>
   <addaction name="actionPrint"/>
   <addaction name="actionPDF"/>
   <addaction name="separator"/>
   <addaction name="actionleft"/>
   <addaction name="actioncenter"/>
   <addaction name="actionright"/>
   <addaction name="actionjustify"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建文件</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/asave.png</normaloff>:/images/asave.png</iconset>
   </property>
   <property name="text">
    <string>另存为...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/quit.png</normaloff>:/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>恢复</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/fileprint.png</normaloff>:/images/fileprint.png</iconset>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
  </action>
  <action name="actionPrintPreview">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/fileprint_preview.png</normaloff>:/images/fileprint_preview.png</iconset>
   </property>
   <property name="text">
    <string>打印预览</string>
   </property>
  </action>
  <action name="actionPDF">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/exportpdf.png</normaloff>:/images/exportpdf.png</iconset>
   </property>
   <property name="text">
    <string>导出PDF</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/stock_delete.png</normaloff>:/images/stock_delete.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/edit-select-all.png</normaloff>:/images/edit-select-all.png</iconset>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/editclear.png</normaloff>:/images/editclear.png</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/find.png</normaloff>:/images/find.png</iconset>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/replace.png</normaloff>:/images/replace.png</iconset>
   </property>
   <property name="text">
    <string>替换</string>
   </property>
  </action>
  <action name="actionzoomIn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/zoom_in.png</normaloff>:/images/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="actionzoomOut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/zoom_out.png</normaloff>:/images/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="actionSwitch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>自动换行</string>
   </property>
  </action>
  <action name="actionToobarShow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>工具栏</string>
   </property>
  </action>
  <action name="actionStatusBarShow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>状态栏</string>
   </property>
  </action>
  <action name="actionHighLightShow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>高亮显示</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/font.png</normaloff>:/images/font.png</iconset>
   </property>
   <property name="text">
    <string>字体设置</string>
   </property>
  </action>
  <action name="actionFontColor">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/textcolor.png</normaloff>:/images/textcolor.png</iconset>
   </property>
   <property name="text">
    <string>字体颜色</string>
   </property>
  </action>
  <action name="actionleft">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/textleft.png</normaloff>:/images/textleft.png</iconset>
   </property>
   <property name="text">
    <string>左对齐</string>
   </property>
  </action>
  <action name="actioncenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/textcenter.png</normaloff>:/images/textcenter.png</iconset>
   </property>
   <property name="text">
    <string>居中对齐</string>
   </property>
  </action>
  <action name="actionright">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/textright.png</normaloff>:/images/textright.png</iconset>
   </property>
   <property name="text">
    <string>右对齐</string>
   </property>
  </action>
  <action name="actionjustify">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/textjustify.png</normaloff>:/images/textjustify.png</iconset>
   </property>
   <property name="text">
    <string>两端对齐</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>平铺</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>层叠</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="text">
    <string>上一窗口</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>下一窗口</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>关闭当前</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>关闭所有</string>
   </property>
  </action>
  <action name="actiontabbed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>标签模式</string>
   </property>
  </action>
  <action name="actionwindow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>窗口模式</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
